spec/factories/curator/metastreams/workflows.rb

```diff
@@ -1,9 +1,11 @@
+# frozen_string_literal: true
+
 FactoryBot.define do
   factory :curator_metastreams_workflow, class: 'Curator::Metastreams::Workflow' do
     association :workflowable, factory: :curator_digital_object
     publishing_state { 1 }
     processing_state { 1 }
-    ingest_origin { "MY_FILE.txt" }
+    ingest_origin { Faker::File.file_name(ext: 'txt') }
```

Comment:
This should be a spreadsheet name, not a filename

Reply:
Should I add a validation or call back for stripping out the extension on create/ update?